_.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+   // Use IntelliSense to find out which attributes exist for C# debugging
```

Comment:
Can you explain why this file is here, and not in `.vscode`? I'm not familiar with the significance of the leading underscore.